src/database/UIDatabase.js

```diff
@@ -15,13 +15,14 @@ const { THIS_STORE_NAME_ID } = SETTINGS_KEYS;
 
 const translateToCoreDatabaseType = type => {
   switch (type) {
+    case 'CashTransaction':
+    case 'CustomerCredit':
     case 'CustomerInvoice':
+    case 'CustomerTransaction':
+    case 'Payment':
     case 'Prescription':
-    case 'SupplierInvoice':
     case 'Receipt':
-    case 'CustomerCredit':
-    case 'Payment':
-    case 'CashTransaction':
+    case 'SupplierInvoice':
```

Comment:
Just a sneaky alphabetical ordering, pretend its not here.

![giphy](https://user-images.githubusercontent.com/43223637/79201760-a7906200-7e8c-11ea-8686-b0ff16664851.gif)